templates/zerver/app/index.html

```diff
@@ -205,7 +205,7 @@
                         </div>
                         <div id="typing_notifications">
                         </div>
-                        <div id="mark_as_read_turned_off_banner" class="alert home-error-bar">
+                        <div id="unread_banner">
```

Comment:
Maybe a better name would be `mark_read_on_scroll_state_banner` -- that covers the range of use cases we have, but isn't so generic that it could sound like a thing that appears in other parts of the app.